CHANGELOG.md

```diff
@@ -1,5 +1,23 @@
 # Mobile Center SDK for iOS Change Log
 
+
+## Version 0.4.2
+
+This version has features that are related to the quality of the provided crash reports as well as a lot of improvements under the hood across all parts of the SDK.
```

Comment:
I would remove "a lot of" in this sentence. It feels a bit of familiar phrasing to me. It's up to you.